src/components/UnitToggle.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ToggleSwitch from '@trendmicro/react-toggle-switch';
+
+const UnitToggle = props => (
+  <div>
+    F
+    <ToggleSwitch
+      size="sm"
+      checked={props.checkUnit}
+      onChange={() => {
```

Comment:
you can do `onChange={unitChange}` (if you destructure props)

Reply:
done